common/config/azure-pipelines/jobs/fast-ci.yaml

```diff
@@ -6,6 +6,7 @@ trigger:
 
 variables:
   - group: Rush Build Cache SAS Token
+  - group: Travis Cobbs - GitHub PAT
```

Comment:
whats this for?

Reply:
In order to download the android native library from GitHub, a personal access token is required. This is completely stupid due to the thing being downloaded being totally public, but that's how GitHub works, and all indications are that they have no intention of changing.

Reply:
instead of creating a new var group for your pat, you couldnt just reuse an exisiting one?
is there elevated priveleges it needs or something like that?

Reply:
The only one I can see that _might_ work is `PerfTesting-Github-AccessToken`. I don't know that it will work, and even if it does, it seems inappropriate to use it based on its name. Using it does require permission from its creator (you). Variable groups have to have unique names, so I included my own name as part of it to make it unique (similar to `Caleb - iTwin GitHub Token`, which based on its description is **not** a token that will work here). If `PerfTesting-Github-AccessToken` is a GitHub PAT with read:packages permission and you really want me to switch to it, I can do so. However, IMO if the desire is for one token to be shared by multiple pipelines, then the token name should reflect what it contains, not the first pipeline to use it.

Reply:
Oh, and the description of `PerfTesting-Github-AccessToken` states that it is already expired. I'm not sure if the description is out of date or if the token is really expired.

Reply:
`common/config/azure-pipelines/jobs/version-bump.yaml` uses `GITHUBTOKEN` which i believe is generated by our admin service account, `imodeljs-admin@users.noreply.github.com`. i would try to use that if possible

Reply:
nvm, that one looks hardcoded to the pipeline